models/stg_recharge__subscription_history.sql

```diff
@@ -32,14 +32,14 @@ final as (
         price,
         quantity,
         status,
-        property_charge_interval_unit_type as charge_interval_unit_type,
+        property_charge_interval_unit_type as charge_interval_unit_type, 
         charge_interval_frequency,
         order_interval_unit,
         order_interval_frequency,
         order_day_of_month,
         order_day_of_week,
-        property_shipping_interval_unit_type as shipping_interval_unit_type,
-        property_shipping_interval_frequency as shipping_interval_frequency,
+        property_shipping_interval_unit_type as shipping_interval_unit_type, 
+        property_shipping_interval_frequency as shipping_interval_frequency, 
```

Comment:
Same comment about these all being custom fields. We should let the variables handle these.

Reply:
Removed.